### DISCUSSION TITLE
How to force WinGet to use English as the output language?

### DISCUSSION THREAD

**Discussion by onyx-nxt** [microsoft/winget-cli#3461] [General]
Is it possible to change the Winget language? I'm working on a script that works with the winget's output, but the output depends on the system language. Can I somehow tell Winget that it should use English as the language without changing it for the system?

**Comment by denelon** [comment]
You might want to try the PowerShell module instead of the CLI. We don't have a locale override for the resource string output in WinGet.

**Comment by Yves848** [comment]
I'm looking for the same thing.
The only wat I figured is to parse the resources file from github and make my own dictionnary.
Take a look at my install.ps1 script on my repo https://github.com/Yves848/WingetPosh
